src/models/utils.js

```diff
@@ -79,11 +79,24 @@ function formatDateString(date) {
 const formatDates = (date) =>
   moment(date, 'ddd MMM DD YYYY HH:mm:ss').format('LL');
 
+// Fix country names so it get return the correct alpha2 code for the flags
+// todo other faulty country names should be added later
+const fixCountryNames = (countries) => countries.map((country) => {
+    if (country.name === 'Tanzania') {
+      return { ...country, name: 'Tanzania, United Republic of' };
```

Comment:
@VWRoli Tanzania, United Republic of this is a standard country name? Interesting.

Reply:
I would assume it is to keep the alphabetical order of the country names.

Reply:
make sense